protobuf/xain_fl/grpc/coordinator.proto

```diff
@@ -55,22 +55,15 @@ message StartTrainingReply {
 
 message EndTrainingRequest {
   ThetaUpdate theta_update = 1;
-  map<string, HistoryValue> history = 2;
-  Metrics metrics = 3;
+  map<string, Metrics> metric = 2;
```

Comment:
`metrics` instead of `metric`, since it is a dictionary of multiple metrics

Reply:
Done.